README.md

```diff
@@ -226,13 +233,21 @@ Returns a `Promise` which resolves to an array of search result `Ad` objects.
 const kijiji = require("kijiji-scraper");
 
 let options = {
-    minResults: 40
+    minResults: 40 // Use only multiples of 20 as Kijiji renders pages in units of 20 listings per page
 };
 
 let params = {
-    locationId: 1700185,  // Same as kijiji.locations.ONTARIO.OTTAWA_GATINEAU_AREA.OTTAWA
-    categoryId: 27,  // Same as kijiji.categories.CARS_AND_VEHICLES
-    sortByName: "priceAsc"  // Show the cheapest listings first
+    locationId: 1700185,  // REQUIRED. Same as kijiji.locations.ONTARIO.OTTAWA_GATINEAU_AREA.OTTAWA
+    categoryId: 27,  // REQUIRED. Same as kijiji.categories.CARS_AND_VEHICLES
+    sortByName: "priceAsc",  // OPTIONAL. Show the cheapest listings first. Default is "dateDesc" i.e. Latest Ads First.
+    maxPrice: 750, // OPTIONAL. Max price is $750 in this example
+    address: "M5R1M3", // OPTIONAL. Postal code (Use without whitespaces)
+    radius: 10, // OPTIONAL. Max distance from address to consider as filter while retrieving ads    
+    adType: "OFFER", // OPTIONAL. Offer Type as provided on Kijiji.ca (Choices are: OFFER and WANTED). By default, it is any i.e. both OFFER and WANTED    
+    hasImages: true, // OPTIONAL. Set it to 1 to avoid returning Ads without Images as shown on Kijiji.ca
+    "attributeMap[furnished_s]": "[1]", // OPTIONAL. Used when looking for apartments. By default, it is any i.e. No preference.
+    "attributeMap[petsallowed_s]": "[1]", // OPTIONAL. Used when looking for apartments. By default, it is any i.e. No preference.
```

Comment:
`attributeMap[furnished_s]` and `attributeMap[petsallowed_s]` no longer work. Likely other `attributeMap` properties too. See #20 / https://github.com/mwpenny/kijiji-scraper/issues/20#issuecomment-521835959. It seems Kijiji removed a bunch of filtering :(

Could you remove these from the examples? Thanks